test/test_problems_mortar_3d.jl

```diff
@@ -7,32 +7,38 @@ using JuliaFEM.Postprocess
 using JuliaFEM.Testing
 using JuliaFEM.Abaqus: create_surface_elements
 
-@testset "test that interface transfers constant field without error" begin
-    meshfile = Pkg.dir("JuliaFEM") * "/test/testdata/block_3d.med"
-    mesh = aster_read_mesh(meshfile)
+### temperature patch tests, sl tet4, dl tet4, sl tet10, dl tet 10
 
-    upper = Problem(Heat, "upper", 1)
+@testset "patch test temperature + abaqus inp + tet4" begin
+    meshfile = Pkg.dir("JuliaFEM") * "/test/testdata/test_problems_mortar_3d_tet4.inp"
```

Comment:
I think you are referring to this: https://discourse.julialang.org/t/rfc-test-file-organization-proposal/1170